code/modules/shuttle/shuttle.dm

```diff
@@ -290,10 +290,8 @@
 /obj/docking_port/mobile/Initialize(mapload)
 	. = ..()
 
-	if(!id)
-		id = "[SSshuttle.mobile.len]"
-	if(name == "shuttle")
-		name = "shuttle[SSshuttle.mobile.len]"
+	id = "[SSshuttle.mobile.len]"
+	name = "[name] shuttle[SSshuttle.mobile.len]"
```

Comment:
Ditto

Reply:
how to handle !00% different ID and NAME near shuttles in another way?